paasta_tools/marathon_tools.py

```diff
@@ -750,27 +749,6 @@ def get_classic_services_running_here_for_nerve(soa_dir):
     return classic_services
 
 
-def get_services_running_here_for_nerve(cluster=None, soa_dir=DEFAULT_SOA_DIR):
```

Comment:
We don't need this function anymore?
Or are we doing this inside of synapse tools? (the concatenation of all the frameworks and junk)